backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -237,6 +237,42 @@ public Paged<AccessRequest> GetAccessRequests(int page = 1, int quantity = 10, s
             var accessRequests = query.Skip((page - 1) * quantity).Take(quantity);
             return new Paged<AccessRequest>(accessRequests, page, quantity, query.Count());
         }
+
+        /// <summary>
+        /// Get all the access requests that users have submitted to the system
+        /// </summary>
+        /// <param name="filter"></param>
+        public Paged<AccessRequest> GetAccessRequests(AccessRequestFilter filter)
+        {
+            this.User.ThrowIfNotAuthorized(Permissions.AdminUsers);
+
+            var query = this.Context.AccessRequests
+                .Include(p => p.Agencies)
+                .ThenInclude(p => p.Agency)
+                .Include(p => p.Roles)
+                .ThenInclude(p => p.Role)
+                .Include(p => p.User)
+                .AsNoTracking();
+
+            query = query.Where(request => request.Status == filter.Status);
+
+            if (!string.IsNullOrWhiteSpace(filter.Role))
+                query = query.Where(ar => ar.Roles.Any(r =>
+                    EF.Functions.Like(r.Role.Name, $"%{filter.Role}")));
+
+            if (!string.IsNullOrWhiteSpace(filter.Agency))
+                query = query.Where(ar => ar.Agencies.Any(a =>
+                    EF.Functions.Like(a.Agency.Name, $"%{filter.Agency}")));
```

Comment:
Curious why the LIKE begins with % but doesn't end with %.  I'd think it would be more intuitive if had both, or ended with %.